app/pages/error.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="error">
+    <h1>{{ title }}</h1>
+    <p>Ups! Nikogo tu nie ma!</p>
+    <nuxt-link to="/" class="btn">Przejdź do strony głównej</nuxt-link>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "Error",
+  data() {
+    return {
+      title: "Błąd 404"
```

Comment:
jw. to nie zawsze jest 404

Reply:
w tym przypadku jest to zawsze 404 (to nie layout/error)

Reply:
A to sry